features/networking/sctp.feature

```diff
@@ -28,21 +28,20 @@ Feature: SCTP related scenarios
     Then the step should succeed
     And the pod named "sctpclient" becomes ready
    
+    And I wait up to 60 seconds for the steps to pass:
+    """
     # sctpserver pod start to wait for sctp traffic
     When I run the :exec background client command with:
-      | pod              | sctpserver                                |
-      | namespace        | <%= project.name %>                       |
-      | oc_opts_end      |                                           |
-      | exec_command     | bash                                      |
-      | exec_command_arg | -c                                        |
-      | exec_command_arg | /usr/bin/nc -k -l 30102 --sctp            |
```

Comment:
Why was -k used to begin with?

Reply:
@weliang1 I also see default deny policy is used in the test, seems this policy resides in sctp folder when we have a similar policy YAML file in testdata/networking/networkpolicy/ folder. Any reason we need two YAML files for same policy?

Reply:
> Why was -k used to begin with?

@asood-rh I think that's the old version you looked at

Reply:
> @weliang1 I also see default deny policy is used in the test, seems this policy resides in sctp folder when we have a similar policy YAML file in testdata/networking/networkpolicy/ folder. Any reason we need two YAML files for same policy?

@asood-rh Good catch, update script to use a common policy file from networking/networkpolicy/defaultdeny-v1-semantic.yaml

Reply:
> > @weliang1 I also see default deny policy is used in the test, seems this policy resides in sctp folder when we have a similar policy YAML file in testdata/networking/networkpolicy/ folder. Any reason we need two YAML files for same policy?
> 
> @asood-rh Good catch, update script to use a common policy file from networking/networkpolicy/defaultdeny-v1-semantic.yaml

New testing log: http://file.rdu.redhat.com/~weliang/ovn-log-update